binary_search_networks/search.py

```diff
@@ -23,30 +28,20 @@ def get_output_space(**args):
 	... b: high bound of n
 	... accuracy: a list of recorded accuracies at each recorded ni
 	'''
-	columns = [
-		'train_accuracy', 
-		'val_accuracy', 
-		'test_accuracy', 
-		'area_under_curve', 
-		'precision',
-		'recall',
-		'F1'
-	]
-	exp_data = pd.DataFrame(columns = columns)
+
+	accuracys = []
 	a = 1
 	b = args['n']
-	for ni in tqdm(range(a, b + 1)):
+	for ni in range(a, b + 1):
```

Comment:
Here, we need to report the the training and the testing accuracy.  Just reporting training  accuracy is not good enough. Right now all the figures in the paper displaying the training accuracy and I  think we need to add the testing accuracy plots as well. I am okay not showing the loss plots or the validation accuracies.